sdk/include/psdk/ntdddisk.h

```diff
@@ -464,6 +473,7 @@ typedef struct _PARTITION_INFORMATION_EX {
   LARGE_INTEGER  PartitionLength;
   ULONG  PartitionNumber;
   BOOLEAN  RewritePartition;
+  BOOLEAN  IsServicePartition;
```

Comment:
This seems to be Windows 8+ . @ThFabba , @tkreuzer , if you have some new PSDK/DDK, could you please confirm this?

Reply:
It's conditional on NTDDI_WIN10_RS3